packages/paste-website/src/pages/components/input/index.mdx

```diff
@@ -68,7 +68,7 @@ export const getStaticProps = async () => {
 
 ### About Input
 
-Use an input when you want to allow a user to enter a single line of text. By default, text inputs allow users to enter any type of text (letters, numbers, and symbols).
+Use an Input when you want to allow a user to enter a single line of text. By default, text inputs allow users to enter any type of text (letters, numbers, and symbols).
 You can set the input to more [specific types](#input-types) if needed. It is used with the [Label](/components/label) and [Help Text](/components/help-text) components.
 
 ### Accessibility
```

Comment:
suggestion: Not sure why we were using components here, but you could replace this whole section with markdown (and fix the "Help Text" and "inline error text" links not appearing correctly more easily).